docs/core/tools/project-json-to-csproj.md

```diff
@@ -0,0 +1,648 @@
+  }
+}
+```
+
+```xml
+<ItemGroup>
+  <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.0.0" />
+</ItemGroup>
+```
+
+> [!NOTE]
+> `imports` on tools are not supported in csproj. Tools that need imports will not work with
+the new `Microsoft.NET.Sdk`.
+
+## buildOptions
+
+See also [Files](#files).
+
+### emitEntryPoint
+
+```json
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <OutputType>Exe</OutputType>
+</PropertyGroup>
+```
+
+If `emitEntryPoint` was `false`, the value of `OutputType` is converted to `Library`, which is the default value:
+
+```json
+{
+  "buildOptions": {
+    "emitEntryPoint": false
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <OutputType>Library</OutputType>
+  <!-- or, omit altogether. It defaults to 'Library' -->
+</PropertyGroup>
+```
+
+### keyFile
+
+```json
+{
+  "buildOptions": {
+    "keyFile": "MyKey.snk"
+  }
+}
+```
+
+The `keyFile` element expands to three properties in MSBuild:
+
+```xml
+<PropertyGroup>
+  <AssemblyOriginatorKeyFile>MyKey.snk</AssemblyOriginatorKeyFile>
+  <SignAssembly>true</SignAssembly>
+  <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+</PropertyGroup>
+```
+
+### Other common build options
+
+```json
+{
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "nowarn": ["CS0168", "CS0219"],
+    "xmlDoc": true,
+    "preserveCompilationContext": true,
+    "outputName": "Different.AssemblyName",
+    "debugType": "portable",
+    "allowUnsafe": true,
+    "define": ["TEST", "OTHERCONDITION"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  <NoWarn>$(NoWarn);CS0168;CS0219</NoWarn>
+  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  <PreserveCompilationContext>true</PreserveCompilationContext>
+  <AssemblyName>Different.AssemblyName</AssemblyName>
+  <DebugType>portable</DebugType>
+  <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  <DefineConstants>$(DefineConstants);TEST;OTHERCONDITION</DefineConstants>
+</PropertyGroup>
+```
+
+## packOptions
+
+See also [Files](#files).
+
+### Common pack options
+
+```json
+{
+  "packOptions": {    
+    "tags": ["machine learning", "framework"],
+    "releaseNotes": "Version 0.9.12-beta",
+    "iconUrl": "http://numl.net/images/ico.png",
+    "projectUrl": "http://numl.net",
+    "licenseUrl": "https://raw.githubusercontent.com/sethjuarez/numl/master/LICENSE.md",
+    "requireLicenseAcceptance": false,
+    "repository": {
+      "type": "git",
+      "url": "https://raw.githubusercontent.com/sethjuarez/numl"
+    },
+    "summary": "numl is a machine learning library intended to ease the use of using standard modeling techniques for both prediction and clustering.",
+    "owners": ["Seth Juarez"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <Title>numl is a machine learning library intended to ease the use of using standard modeling techniques for both prediction and clustering.</Title>
```

Comment:
No, your comments didn't cause confusion. It was a leftover change I forgot to edit. Thanks for your review @natemcmaster!